app/jobs/bcc_post.rb

```diff
@@ -21,12 +21,14 @@ def execute(args)
 
   def send_to(targets, targets_key, params, sender)
     targets.each do |target|
-      temp_params = params
+      temp_params = params.clone
```

Comment:
Instead of `clone` could you do something like:

`temp_params["raw"].gsub!(/%{username}/i, target)`

That should modify the string in place (which is faster).

Reply:
I don't follow. The whole issue (for the original PR) was that not cloning the parameters resulted in the original parameters being changed instead of temporary ones.